### PR TITLE
Move devtools_flutter code into package:devtools_app

### PR DESCRIPTION
**Pull request by DaveShuckerow**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -35,4 +42,4 @@
 			"program": "${file}"
 		}
 	]
-}
+}
```
Comment (jacob314) [inline .vscode/launch.json]: add back trailing whitespace

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,32 +10,78 @@ You can do this online, and it only takes a minute. If you've never submitted co
...
 - `cd devtools/packages/devtools_app`
-- `pub get`
+- `flutter pub get`
 
 From a separate terminal:
 - `cd <path/to/flutter-sdk>/examples/flutter_gallery`
 - ensure the iOS Simulator is open (or a physical device is connected)
 - `flutter run`
 
+### Pure-Dart Web App
```
Comment (jacob314) [inline CONTRIBUTING.md]: nit: call this dart:html Web App or "Legacy dart:html Web App"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,32 +10,78 @@ You can do this online, and it only takes a minute. If you've never submitted co
...
+### Where to put Flutter code
+
+We also roll DevTools' code into a bazel build system, where we need to run only the non-flutter
+code from the app. To facilitate this, please keep code that imports package:flutter under a
+directory named 'flutter'.
+
+For example, `lib/src/flutter`, `lib/src/inspector/flutter`, or `test/flutter` are all acceptable
+locations to put new Flutter code.
+
+When we are ready to turn down the pure-Dart web version of the app, we can delete all code that
```
Comment (jacob314) [inline CONTRIBUTING.md]: can --> will

File: .vscode/launch.json (hunk 1 above)
Comment (DaveShuckerow) [reply .vscode/launch.json]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (DaveShuckerow) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 3 above)
Comment (DaveShuckerow) [reply CONTRIBUTING.md]: Done.

**Pull request merged**